roles/populate_aap/defaults/main.yml

```diff
@@ -36,8 +38,8 @@ populate_aap_credentials:
     organization: "{{ populate_aap_organization_name }}"
     credential_type: Machine
     inputs:
-      username: user
-      ssh_key_data: "{{ lookup('file', '/home/user/.ssh/id_rsa') }}"
+      username: "{{ populate_aap_default_host_user }}"
+      ssh_key_data: "{{ lookup('file', populate_aap_ssh_key_path) if populate_aap_ssh_key_path is defined else '' }}"
```

Comment:
This should be just the var {{ populate_aap_ssh_key_path }} and the role should contain all of the non variable information and then this would close the other issue #213 
This is a best practice that the defaults should be vars not logic.